commons-jxpath/src/main/java/org/apache/commons/jxpath/ri/compiler/ExtensionFunction.java

```diff
@@ -69,7 +69,7 @@ public String toString() {
         final Expression[] args = getArguments();
         if (args != null) {
             for (int i = 0; i < args.length; i++) {
-                if (i >= 0) {                               // <change7>
+                if (i > 0) {                               
```

Comment:
Should be a formatting change and not logical change

Reply:
@Mashequr please fix in the next 2-3 days as otherwise we'll have to assign to someone else.